src/nxdoc/nuxeo-server/upgrading-the-nuxeo-platform/upgrade-from-lts-2016-following-fast-tracks.md

```diff
@@ -400,3 +400,87 @@ Nuxeo Segment.io plugin is now aligned on `com.github.segmentio:analytics:1.0.7`
 - [Release notes for 9.2]({{page version='' space='nxdoc' page='nuxeo-server-release-notes'}})
 
 {{! /multiexcerpt}}
+
+
+{{! multiexcerpt name='9.2-to-9.3-upgrade-page'}}
+## From 9.2 to 9.3
+
+### Installation and Configuration
+
+{{! multiexcerpt name='upgrade-8.10-installation-elasticsearch-upgrade'}}
+#### Elasticsearch Upgrade
+
+A new version of Elasticsearch is required, here are the necessary steps to upgrade:
+- Upgrade Elasticsearch version
+- Update your custom settings and mapping to follow new Elasticsearch rules
+- Re-index the repository to apply the new settings and mapping
+- Update your custom code that query Elasticsearch directly
+
+** Upgrade Elasticsearch Version **
```

Comment:
Formatting?

Reply:
this formating is taken from another upgrade note it should have a meaning for metalsmth